perllib/FixMyStreet/Roles/CobrandBulkyWaste.pm

```diff
@@ -375,15 +375,20 @@ sub _check_within_bulky_refund_window {
 }
 
 sub bulky_nice_collection_date {
-    my ($self, $report_or_date) = @_;
+    my ($self, $report_or_date, $full_date) = @_;
+
     my $dt = do {
         if (ref $report_or_date eq 'FixMyStreet::DB::Result::Problem') {
             $self->collection_date($report_or_date);
         } else {
             $self->_bulky_date_to_dt($report_or_date);
         }
     };
-    return $dt->strftime('%d %B');
+    if ($full_date) {
+        return $dt->strftime('%A %d %B %G');
```

Comment:
Unless you really have some hankering for confusing year numbers when the ISO week number is in a different calendar year, I'd go with `%Y` ;)